scripts/deploy-contracts.sh

```diff
@@ -47,6 +47,12 @@ if [ "${BASE_EXECUTION_CLIENT}" != "geth_dev" ]; then
 fi
 cast rpc evm_setAutomine true --rpc-url $RIVER_ANVIL_RPC_URL
 
+# Only anvil supports setCode but this might be a local geth node
+if [ "${BASE_EXECUTION_CLIENT}" != "geth_dev" ]; then
+    cast rpc anvil_setCode $SEQUENCE_MULTICALL_ADDRESS $SEQUENCE_MULTICALL_BYTECODE --rpc-url $BASE_ANVIL_RPC_URL
+fi
+cast rpc anvil_setCode $SEQUENCE_MULTICALL_ADDRESS $SEQUENCE_MULTICALL_BYTECODE --rpc-url $RIVER_ANVIL_RPC_URL
+
```

Comment:
Both of these lines look the same. Should there be an else, or different code for geth?

Reply:
I mimicked line 45-48. As far as I understand, the River chain is always deployed by Anvil. But the Base chain may be on Anvil or a local Geth.

Reply:
yes you're right. thank you.